src/app/admin/volunteers/shared/newvolunteer-register-form/newvolunteer-register-form.component.html

```diff
@@ -0,0 +1,168 @@
+<div class="modal_header_line">
+  <a>Crează Voluntar</a>
+</div>
+
+<div class="date_personale_div">
+  <div class="date_personale_line">
+    <a>Date personale</a>
+  </div>
+  <form [formGroup]="form" (ngSubmit)="onSubmit($event)">
+    <mat-grid-list cols="2" rowHeight="65px" gutterSize="10px">
+      <div *ngFor="let f of formFields">
+        <mat-grid-tile *ngIf="f.header !== 'Sector'">
+          <div>
+            <mat-grid-tile-header>
+              <a>{{ f.header }}</a>
+            </mat-grid-tile-header>
+            <input
+              [type]="f.type || 'text'"
+              formControlName="{{ f.controlName }}"
+            />
+            <mat-error *ngIf="form.get(f.controlName).invalid && form.touched">
+              {{ f.errorMessage || "atenție, eroare - cîmp obligatoriu!" }}
+            </mat-error>
+          </div>
+        </mat-grid-tile>
+
+        <mat-grid-tile *ngIf="f.header === 'Sector'">
+          <div>
+            <mat-grid-tile-header>
+              <a>{{ f.header }}</a>
+            </mat-grid-tile-header>
+            <mat-select formControlName="{{ f.controlName }}">
+              <mat-option *ngFor="let sec of zones" [value]="sec">
+                {{ sec.charAt(0).toUpperCase() + sec.slice(1) }}
+              </mat-option>
+            </mat-select>
+            <mat-error *ngIf="form.get(f.controlName).invalid && form.touched">
+              {{ f.errorMessage || "atenție, eroare - cîmp obligatoriu!" }}
+            </mat-error>
+          </div>
+        </mat-grid-tile>
+      </div>
+    </mat-grid-list>
+
+    <mat-divider
+      style="color: #ccc; margin-top: 10px; margin-left: -40px"
+    ></mat-divider>
+
+    <div class="date_personale_line" style="margin-top: 28px">
+      <a>Rolul Voluntarului</a>
+    </div>
+
+    <mat-form-field>
+      <mat-select formControlName="role" required multiple>
+        <mat-option
+          *ngFor="let role of roles | keyvalue: enumUnsorted"
+          [value]="role.key"
+        >
+          <img
+            src="/assets/icons/volunteers/{{
+              volunteerRolesIncons[role.key]
+            }}.svg"
```

Comment:
Svg icons should be registered in `app.component.ts` and used properly using `<mat-icon svgIcon="icon-name"></mat-icon>`

Reply:
agree - will change as wrote

Reply:
Done